document/cloud-client/parse_with_model.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START documentai_parse_with_model]
+from google.cloud import documentai_v1beta2 as documentai
+
+
+def parse_with_model(
+        project_id='YOUR_PROJECT_ID',
+        input_uri='gs://cloud-samples-data/documentai/invoice.pdf',
+        automl_model='YOUR_AUTOML_MODEL_NAME_HERE'):
```

Comment:
```suggestion
        model_id='YOUR_AUTOML_MODEL_ID'):
```

Reply:
It actually does need to be a resource name rather than an ID.